### PR TITLE
Add semicolon to asserts that were missing it.

### PR DESCRIPTION
**Pull request by gassmoeller**
This is like dealii/dealii#16037, but for ASPECT to make the tester compile with deal.II dev.

### CODE REVIEW

**Comment by gassmoeller** [comment]
Necessary because of dealii/dealii#16026

**Comment by gassmoeller** [comment]
Oh, wait. So the new form is not compatible with the old one before deal.II 9.6.0.pre? That will make it a bit more complicated. I guess I need to introduce braces around all the if-else branches?

**Comment by tjhei** [comment]
Yes, you will need braces around it. Considering this is a macro, it is safer anyways.

**Review by tjhei — APPROVED** [review]
Thanks for fixing.

File: tests/solidus_initial_conditions.cc [hunk 1]
```diff
@@ -208,8 +206,8 @@ namespace aspect
                     p*=1.e3;                    // km to meters
                   else if (P_Unit!="m")
                     AssertThrow(false,ExcMessage ("Unit of the second column of melting curve data "
-                                                  "has to be one of the following: Pa/GPa/km/m."))
-                  }
+                                                  "has to be one of the following: Pa/GPa/km/m."));
+                }
```
Comment (tjhei) [inline tests/solidus_initial_conditions.cc:210]: another reason for fixing this is the previously incorrect indentation. :-)

**Pull request merged**